docs/conversion_examples_gallery/recording_and_sorting/neuroscope.rst

```diff
@@ -0,0 +1,29 @@
+Neuroscope data conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example of how to convert ``Neuroscope`` to nwb
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from nwb_conversion_tools import NeuroscopeRecordingInterface
+
+Then we creates a datetime object representing the date of the first of January of 2020 in the US/Pacific time-zone. 
+We use this for the metadata:
+    
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))    
+    >>> metadata = {"NWBFile":{"session_start_time":session_start_time}}
```

Comment:
add a .get_metadata command

Reply:
Yes, I did not push this. Question:
Would you like to have the output of conversion.get_metadata() printed and explained or would you rather just have the call there and nothing else?

Reply:
See the new file for a comparison.